src/webrtc/call.ts

```diff
@@ -1751,6 +1747,16 @@ export class MatrixCall extends EventEmitter {
         logger.debug(`stopAllMedia (stream=${this.localUsermediaStream})`);
 
         for (const feed of this.feeds) {
+            if (feed.isLocal()) {
+                if (feed.purpose === SDPStreamMetadataPurpose.Usermedia) {
+                    this.client.getMediaHandler().stopUserMediaStream(feed.stream);
+                    continue;
```

Comment:
The `continue` doesn't seem to be necessary here?

Reply:
Do we need to cover unknown `feed.purpose` values? That's the only reason I did it. It avoids calling `stop()` on the tracks multiple times.

Reply:
I think the new version is a bit better

Reply:
Oops, I missed a part of the code, the current solution seems great 👍